src/main/resources/org/terasology/launcher/bundle/LabelsBundle.properties

```diff
@@ -75,6 +75,7 @@ message_error_loadSettings=
 message_error_operatingSystem=
 message_error_storeSettings=
 message_error_tempDirectory=
+message_error_cacheDirectory=
```

Comment:
need to add actual text somewhere 🙄 double check if there is an error label w.r.t. to `download` directory.